backend/app/db/alembic_dev/versions/0466229c861e_add_user_answer.py

```
@@ -0,0 +1,45 @@
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('exercise_user_answers',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('exercise_id', sa.Integer(), nullable=False),
+    sa.Column('user_id', sa.String(length=128), nullable=False),
+    sa.Column('answer', sa.TEXT(), nullable=False),
+    sa.Column('scoring_results', sa.TEXT(), nullable=True),
+    sa.Column('created_at', sa.DateTime(), nullable=False),
+    sa.ForeignKeyConstraint(['exercise_id'], ['exercises.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_index(op.f('ix_exercise_user_answers_exercise_id'), 'exercise_user_answers', ['exercise_id'], unique=False)
```

Ruff (0.8.2)

Line too long (121 > 100)

(E501)